tests/acceptance/profiles.tsx

```diff
@@ -15,6 +16,21 @@ const addUserToRole = new TypedMethod<{ userId: string, scope: string, role: str
 const createHunt = new TypedMethod<{ name: string }, string>('test.methods.profiles.createHunt');
 const joinHunt = new TypedMethod<{ huntId: string, userId: string }, void>('test.methods.profiles.joinHunt');
 
+const subscribeAsync = (name: string, ...args: any[]) => new Promise<Meteor.SubscriptionHandle>(
+  (resolve, reject) => {
+    const handle = Meteor.subscribe(name, ...args, {
+      onError: (reason?: Meteor.Error) => {
```

Comment:
Per https://docs.meteor.com/api/pubsub.html#Meteor-subscribe there is no `onError`, only `onStop` (which fires with no args on client-driven teardown and a Meteor.Error if the publish function throws or the server cancels the sub).